### PR TITLE
Implement Slash Commands

### PR DESCRIPTION
**Pull request by RisingFog**
This PR implements the majority of the commands as slash commands, including the main verify command. If the $verify command is sent as a DM, it will inform the user to use the slash command version of verify instead. All slash command replies are now ephemeral, which means that only the user who initiates the command will see the reply from the bot.

image

image

image

Additionally, with the move to slash commands, it now allows users to autofill certain parameters (such as log channel and role selection) for commands which require it.

image

The move to slash commands makes the bot much easier to use for both users wishing to verify their memberships, as well as moderators and administrators wanting to manage their server.

### CODE REVIEW

File: settings.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # External
+from xmlrpc.client import Boolean
```
Comment (nonJerry) [inline settings.py]: I think import wasn't intentionally?

File: utility.py [hunk 2]
```diff
@@ -1,4 +1,5 @@
 #External
+from random import paretovariate
```
Comment (nonJerry) [inline utility.py]: Again probably not intentional import I guess XD

File: settings.py (hunk 1 above)
Comment (RisingFog) [reply settings.py]: Nope

File: utility.py (hunk 2 above)
Comment (RisingFog) [reply utility.py]: I don’t even remember adding these

**Review by nonJerry — APPROVED** [review]

**Pull request merged**